use-cases/rag-on-gke/frontend/src/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12.4-slim
```

Comment:
Same comment as above.

Reply:
Yes , I just used the version which was already there in fine-tuned model for deployments . I can switch to this image.

Reply:
fixed